workflows/computational-chemistry/fragment-based-docking-scoring/fragment-based-docking-scoring-tests.yml

```diff
@@ -21,10 +21,11 @@
     SuCOS threshold: 0
   outputs:
     Scored and filtered poses:
-      class: File
-      assert:
-        has_line: "$$$$"
-        n: 4
-      assert:
-        has_test: "SuCOS_Score"
-        n: 4
+      asserts:
+        has_line:
+          line: "$$$$"
+          n: 4
+      asserts:
+        has_text:
+          text: "SuCOS_Score"
+          n: 4
```

Comment:
I think this will only test the last one, do you have a check for this?

Reply:
I don't think that's really checkable as that's a yaml language issue, we don't go that deep.